src/common/AndroidUtils.ts

```diff
@@ -458,11 +458,14 @@ export class AndroidUtils {
                         );
                     });
                     child.stderr.on('data', (data) => {
-                        reject(
-                            new Error(
-                                `Could not create emulator. Command failed: ${createAvdCommand}\n${data}`
-                            )
-                        );
+                        const msg = data.toString() as string;
+                        if (msg && msg.includes('Error:')) {
+                            reject(
+                                new Error(
+                                    `Could not create emulator. Command failed: ${createAvdCommand}\n${data}`
+                                )
+                            );
+                        }
```

Comment:
For some reason ADB also send non-errors (such as `Warning:`) to the `stderr` too 🤦  So we explicitly check for `Error:` to see if it is truly an error or not.